content/en/blog/_posts/2020-08-21-Moving-Forward-From-Beta/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: blog
+title: "Moving Forward From Beta"
+date: 2020-08-21
+slug: moving-forward-from-beta
+---
+
+**Author**: Tim Bannister, The Scale Factory
+
+In Kubernetes, features follow a defined
+[lifecycle](/docs/reference/command-line-tools-reference/feature-gates/#feature-stages).
+First, as the twinkle of an eye in an interested developer. Maybe, then,
+sketched in online discussions, drawn on the online equivalent of a cafe
+napkin. This rough work typically becomes a
+[Kubernetes Enhancement Proposal](https://github.com/kubernetes/enhancements/blob/master/keps/0001-kubernetes-enhancement-proposal-process.md#kubernetes-enhancement-proposal-process) (KEP), and
+from there it usually turns into code.
+
+For Kubernetes v1.20 and onwards, we're focusing on helping that code
+graduate into stable features.
+
+That lifecycle I mentioned runs as follows:
+
+![Alpha → Beta → General Availability](feature_stages.svg)
```

Comment:
Suggestion:
The text for `general availability` is small. You could make that text larger or add `GA` to the stable box such as:
stable/GA

Reply:
@kbhawkey do you know a good way to set that text size? It's set to `2.82222px` but then that size in pixels is scaled based on the `viewBox`. I could make it larger for everyone but then it might overlap the diagram, without extra measures to avoid that happening.

Reply:
Sorry, I do not.

Reply:
I've had a go - how does the preview look now?

Reply:
I think it looks better.